tools/unitctl/README.md

```diff
@@ -51,11 +51,44 @@ desired.
 
 ## Features (Current)
 
+```
+CLI interface to the NGINX UNIT Control API
+
+Usage: unitctl [OPTIONS] <COMMAND>
+
+Commands:
+  instances  List all running UNIT processes
+  edit       Open current UNIT configuration in editor
+  import     Import configuration from a directory
+  execute    Sends raw JSON payload to UNIT
+  status     Get the current status of UNIT
+  listeners  List active listeners
+  help       Print this message or the help of the given subcommand(s)
+
+Options:
+  -s, --control-socket-address <CONTROL_SOCKET_ADDRESS>
+          Path (unix:/var/run/unit/control.sock), tcp address with port (127.0.0.1:80), or URL
+  -w, --wait-timeout-seconds <WAIT_TIME_SECONDS>
+          Number of seconds to wait for control socket to become available
+  -t, --wait-max-tries <WAIT_MAX_TRIES>
+          Number of times to try to access control socket when waiting [default: 3]
+  -h, --help
+          Print help
+  -V, --version
```

Comment:
Hm, is this a capital V for a reason? A lot of cli tools use `-v` for the short flag, lowercase V.

Reply:
I'm assuming we're reserving `-v` for `--verbose`

Reply:
I am not sure, this was done in the original code from @dekobon

Reply:
I chose the short flag to be capitol `V` to align with nginx and unit.